client/web/web.go

```diff
@@ -703,13 +713,20 @@ func (s *Server) serveGetNodeData(w http.ResponseWriter, r *http.Request) {
 }
 
 func availableFeatures() map[string]bool {
-	return map[string]bool{
+	features := map[string]bool{
 		"advertise-exit-node": true,                            // available on all platforms
 		"advertise-routes":    true,                            // available on all platforms
 		"use-exit-node":       distro.Get() != distro.Synology, // see https://github.com/tailscale/tailscale/issues/1995
 		"ssh":                 envknob.CanRunTailscaleSSH() == nil,
 		"auto-update":         version.IsUnstableBuild() && clientupdate.CanAutoUpdate(),
 	}
+	if hostinfo.GetEnvType() == hostinfo.HomeAssistantAddOn {
+		// Setting SSH on Home Assistant causes trouble on startup
+		// (since the flag is not being passed to `tailscale up`).
+		// It's also not terribly useful since it's running in a separate container.
```

Comment:
```suggestion
		// Setting SSH on Home Assistant causes trouble on startup
		// (since the flag is not being passed to `tailscale up`).
		// Although Tailscale SSH works here, it's not terribly useful since it's running in a separate container.
```

just calling out why it's not disabled from `envknob.CanRunTailscaleSSH()`

Reply:
done